src/main.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from typing import List
+
+app = FastAPI()
+
+# Create
+# Boolean → bool
+# "," ";" は不要
+class CreateTodoList(BaseModel):
+    key: str
+    check: bool | None = None
+    text: str | None = None
```

Comment:
APIの関数を見るに、おそらくlistで受け取りたいのだろう
ということでこう書くと出来ます

class TodoList は CreateTodoListの中にネストして定義することも出来ますが、
おそらく別のAPIでも使い回すので、外で定義したほうが便利そう

```suggestion
class TodoList(BaseModel):
    key: str
    check: bool | None = None
    text: str | None = None
    
class CreateTodoList(BaseModel):
    todo_lists: list[TodoList]
```

Reply:
なるほど…！
ありがとうございます。

ちなみになんですが、外で定義するとなるとどういったフォルダで管理するのがよかったりするのでしょうか？

Reply:
src/schemasとかですかね？僕もあんまりわかっていない
https://zenn.dev/dencyu/articles/a94928b9ce45f7